bootstrap.py

```diff
@@ -285,11 +285,11 @@ def clean_environment():
 
 
 def generate_katello_facts():
-    if FQDN.find(".") != -1 and not os.path.exists('/etc/rhsm/facts/katello.facts'):
-        print_generic("Workaround for FQDN")
-        katellofacts = open('/etc/rhsm/facts/katello.facts', 'w')
-        katellofacts.write('{"network.hostname":"%s"}\n' % (FQDN))
-        katellofacts.close()
+    """Write katello_facts file based on FQDN"""
+    print_generic("Writing FQDN katello-fact")
+    katellofacts = open('/etc/rhsm/facts/katello.facts', 'w')
+    katellofacts.write('{"network.hostname-override":"%s"}\n' % (FQDN))
```

Comment:
From @jlsherrill: "katello will prefer using network.fqdn,  network.hostname-override, and finally (if none of those are present) network.hostname" I changed our to using network.hostname-override from network.hostname (which we used previously) to be consistent with the katello-ca-consumer RPM.